ontology/envo.md

```diff
@@ -52,6 +52,7 @@ products:
 publications:
 - id: https://www.ncbi.nlm.nih.gov/pubmed/24330602
```

Comment:
```suggestion
- id: https://dx.doi.org/10.1186/2041-1480-4-43
```

Reply:
dx.doi.org is not an allowed prefix, can we change to doi.org? (works just was well)

https://github.com/OBOFoundry/OBOFoundry.github.io/blob/10dd17d93ef4537cbab02e35a3c867e1ab100c36/tests/test_integrity.py#L23

https://doi.org/10.1186/2041-1480-4-43